job-service-db/src/main/resources/procedures/changelog.xml

```diff
@@ -295,4 +295,16 @@
                          schemaName="public">
         </createProcedure>
     </changeSet>
+    <changeSet id="drop_deleted_task_tables_proc" runOnChange="true" author="Gamma Team">
+        <createProcedure path="procedures/dropTables.sql"
```

Comment:
The procedure filename should align with the actual name.

Reply:
Renamed